src/repl/repl.ts

```diff
@@ -35,7 +36,9 @@ function startRepl(
           eval: (cmd, unusedContext, unusedFilename, callback) => {
             runInContext(cmd, context, options).then(obj => {
               if (obj.status === 'finished' || obj.status === 'suspended-non-det') {
-                callback(null, obj.value)
+                // if the result is a representation, display the representation.
+                // else, default to standard value representation.
+                callback(null, obj.representation ? obj.representation : obj.value)
```

Comment:
Should we check for `!==` undefined here instead of truthy/falsy?

Reply:
Will do!